cmd/login/runner.go

```diff
@@ -29,13 +29,13 @@ type runner struct {
 }
 
 type LoginOptions struct {
-	isWCClientCert            bool
-	selfContained             bool
-	selfContainedClientCert   bool
-	switchToContext           bool
-	switchToClientCertContext bool
-	originContext             string
-	contextOverride           string
+	selfContained     bool
+	selfContainedWC   bool
+	isWC              bool
+	switchToContext   bool
+	switchToWCContext bool
+	originContext     string
+	contextOverride   string
```

Comment:
the naming here was misleading so i corrected the flags to what they are really used for

Reply:
`isWCClientCert` ->`isWC`
`selfContainedClientCert` -> `selfContainedWC`|
`switchToClientCertContext` - > `switchToWCContext`

Reply:
I think we should keep the `clientCert` within the name since there is also the case of OIDC on a workload cluster which would not require these options.

Reply:
But none of the options are actually tied to a clientCert vs OIDC decision-making, they are purely decision-making between WC and MC

Reply:
So even with OIDC WC these are valid options: 

`selfContainedWC` self-contained kubeconfig is still a valid  option for both OIDC or clientcert 
`switchToWCContext`  switching to context or not is still valid for both OIDC or client cert


and ATM `kubectl gs` do not have any kind of implementation to support OIDC for WC login, it can only login into MC with OIDC as it needs athena on the cluster

Reply:
I get where you are coming from since it's not the default but we actually have some customers that use kgs to log into their WCs via OIDC.
The difference is that in case of WC client-cert we go the following route:
log into MC via OIDC to get resources from MC -> log into WC via client-cert
So there is an extra context switch in the middle we need to be mindful of in case of self-contained contexts and keeping the initial context selected.
Functionally nothing changes by naming the options differently, but by equating this to `WC` case we make the code even harder to read than it already is since in some cases we log into a WC and yet the option will be false.

In case of WC OIDC we do not need to log into the MC first since we do not need any resources from there and can log into the WC directly. It is kinda the same as logging into MC other than that the flags might contain both MC name + WC name for selecting this context. 

It seems that the EKS case is kind of in the middle. While we do need to log into the MC first, we are not creating a client certificate. :thinking: So I guess you are right in that the name does not apply. I can't think of a great name but, how about `secondContext`?

Reply:
resolved in call